### PR TITLE
docs(navbar): add navbar example with brand, dropdown and toggle

### PR DESCRIPTION
**Pull request by bence444**
Added more examples to the navbar page in the docs. It contains the following examples:
- Navbar with a `NavbarBrandComponent`
- Navbar with `NavbarBrandComponent` and `DropdownComponent`
- Navbar with `NavbarBrandComponent` and `NavbarToggleComponent` to collapse/open navbar on mobile devices

### CODE REVIEW

File: apps/docs/docs/components/navbar/_responsive.component.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  selector: 'flowbite-demo-navbar-responsive',
+  standalone: true,
+  imports: [NavbarComponent, NavbarContentComponent, NavbarItemComponent, NavbarBrandComponent, NavbarToggleComponent],
+  template: `
+<flowbite-navbar>
+  <flowbite-navbar-brand>
+    Flowbite
+  </flowbite-navbar-brand>
+
+  <flowbite-navbar-toggle></flowbite-navbar-toggle>
```
Comment (MGREMY) [inline apps/docs/docs/components/navbar/_responsive.component.ts:15]: Auto close when we can

**Review by MGREMY — APPROVED** [review]

**Comment by MGREMY** [comment]
Waiting for the Vercel preview to be up (we are having a little issue on PR preview with Vercel, so I'll use this one as a test)

Thanks for the PR ! :D

**Comment by SutuSebastian** [comment]
@MGREMY manually deployed the preview and disabled a rule in Vercel, please try again with some new PR and/or commit here

**Comment by MGREMY** [comment]
@SutuSebastian Okay thanks,
We'll try again after the fix commits on my comments :)

File: apps/docs/docs/components/navbar/_brand.component.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { NavbarComponent, NavbarContentComponent, NavbarItemComponent, NavbarBrandComponent } from 'flowbite-angular';
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'flowbite-demo-navbar-brand',
+  standalone: true,
+  imports: [NavbarComponent, NavbarItemComponent, NavbarContentComponent, NavbarBrandComponent],
+  template: `
+<flowbite-navbar [isFixed]="true" [isOpen]="true">
```
Comment (MGREMY) [inline apps/docs/docs/components/navbar/_brand.component.ts:10]: Remove `[isOpen]="true"`

File: apps/docs/docs/components/navbar/_dropdown.component.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { NavbarComponent, NavbarContentComponent, NavbarBrandComponent, NavbarItemComponent, DropdownComponent, DropdownItemComponent } from 'flowbite-angular';
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'flowbite-demo-navbar-dropdown',
+  standalone: true,
+  imports: [NavbarComponent, NavbarContentComponent, NavbarBrandComponent, NavbarItemComponent, DropdownComponent, DropdownItemComponent],
+  template: `
+<flowbite-navbar [isFixed]="true" [isOpen]="true">
```
Comment (MGREMY) [inline apps/docs/docs/components/navbar/_dropdown.component.ts:10]: Remove `[isOpen]="true"`

File: apps/docs/docs/components/navbar/_brand.component.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  <flowbite-navbar-brand>
+    Flowbite
```
Comment (MGREMY) [inline apps/docs/docs/components/navbar/_brand.component.ts:12]: Add the flowbite'-angular's logo as in the app.component.html

**Review by MGREMY — APPROVED** [review]

**Pull request merged**